Level 1. Lesson 3. Выполнил задание 1 с калькулятором.

t88code opened the pull request:
Level 1. Lesson 3. Выполнил задание 1 с калькулятором.
https://gb.ru/lessons/182479/homework

t88code committed:
Create main.go
Lesson 3 is ready

goarchitecture commented on lesson-3/main.go:
```diff
@@ -0,0 +1,154 @@
+
+	for {
+		if err := calculator(); err != nil {
+			fmt.Println(err.Error())
```
лучше вводить в одну строку: сообщение и ошибку

t88code replied:
Готово.

goarchitecture commented on lesson-3/main.go:
```diff
@@ -0,0 +1,154 @@
+
+}
+
+func calculator() error {
```
:+1: за отдельную функцию с возвращаением ошибки

t88code replied:
Спасибо!

goarchitecture commented on lesson-3/main.go:
```diff
@@ -0,0 +1,154 @@
+		return fmt.Errorf("введено неверное значение")
+	}
+
+	if op == "+" || op == "-" || op == "*" || op == "/" {
```
замечание со звёздочкой. 
мы уже прошли массивы и слайсы. Подумайте, как можно переделать этот if - чтобы не было постоянных `II` и чтобы было легче добавлять или убирать операторы

goarchitecture reviewed (commented):
отлично!

t88code committed:
Правки:
1. Реализовано отображение ошибки и сообщения в одной сроке(line 13).
2. Добавлена отдельная функция findOp, которая выполняет поиск операции(string) по слайсу([]string). Мотивация: добавлять операции в слайс удобнее, чем использовать || для каждой операции. Также код становится более читабельным.
